docs/systemd.md

```diff
@@ -55,6 +55,9 @@ Apart from the `.service` file you'll need the service startup script and defaul
 configuration file, they can be found in the [`scripts`](../docker/package/scripts) and
 [`defaults`](../docker/package/defaults) folders respectively.
 
+## Systemd services on WSL
+
+Unfortunately, `systemd` is not officially supported on WSL. However, there are some unofficial workarounds for it.
```

Comment:
This is in the right place and says the right thing IMO :slightly_smiling_face: 

I would only add that:
```suggestion
Unfortunately, `systemd` is not officially supported on WSL.

However, there are several unofficial workarounds for it and some have been known to work with `tezos-packaging`'s units.
If you are successfully running a distro on WSL with `systemd`, the documentation above should apply to you too.
```